cicd/gitops/gitops-release-notes.adoc

```diff
@@ -0,0 +1,21 @@
+//OpenShift Pipelines Release Notes
+include::modules/gitops-document-attributes.adoc[]
+[id="gitops-release-notes"]
+= {gitops-title} release notes
+:context: gitops-release-notes
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+{gitops-title} is a cloud-native CI/CD experience based on the Tekton project which helps you automate the following tasks:
```

Comment:
Please use https://docs.openshift.com/container-platform/4.7/cicd/gitops/understanding-openshift-gitops.html#key-features_understanding-openshift-gitops and https://docs.openshift.com/container-platform/4.7/cicd/gitops/understanding-openshift-gitops.html#about-redhat-openshift-gitops_understanding-openshift-gitops to add a couple of introductory sentences here.

Reply:
Done.